src/QsCompiler/DocumentationParser/DocNamespace.cs

```diff
@@ -185,10 +185,11 @@ void WriteItem(DocItem i)
         }
 
         /// <summary>
-        /// Writes the YAML file for this namespace.
+        /// Writes the YAML file for this namespace to a stream.
         /// </summary>
-        /// <param name="directoryPath">The directory to write the file to</param>
-        internal void WriteToFile(string directoryPath)
+        /// <param name="stream">The stream to write to.</param>
+        /// <param name="rootNode">The preexisting YAML mapping node.</param>
```

Comment:
It might be good to clarify that the value of `rootNode` represents the preexisting contents of a namespace YAML file.